docs/visual-basic/programming-guide/language-features/procedures/snippets/how-to-call-an-event-handler/Class1.vb

```diff
@@ -0,0 +1,27 @@
+Public Class SpecialForm : Inherits System.Windows.Forms.Form
+
+    Sub New()
+        OnFormOpen()
+    End Sub
```

Comment:
@IEvangelist I added the ctor as you suggested. This, however, won't now appear in the article as it's outside the `<Snippet>` tags. Should I include the whole file as the final snippet in the article?

I also strongly recommend re-writing the sample to be more generic. it's currently tied to WinForms while it's supposed to explain a more general thing "Events".

Reply:
Hi @Youssef1313 - thank you for your work on this. It is okay to get things incrementally better, and propose further improvements as we discover these opportunities. I'd prefer concluding this PR first, but I love the ideas.

Reply:
@IEvangelist Agreed. I'll open an issue to track updating the examples to be more generic than WinForms.